CMakeLists.txt

```diff
@@ -0,0 +1,151 @@
+cmake_minimum_required(VERSION 3.8.2)
+project(cglm LANGUAGES C)
+
+set(C_STANDARD 99)
```

Comment:
We have a `-std=gnu11` flag in **makefile.am** file, MSVC may not support C11/C18... of course I'm not sure. What do you suggest?

Reply:
I don't have how to test on MSVC, but I'm think cglm project will compile.